browser/src/control/Control.AboutDialog.ts

```diff
@@ -188,6 +188,10 @@ class AboutDialog {
 			null,
 		);
 		copyversion.setAttribute('id', 'modal-dialog-about-dialog-box-copybutton');
+		copyversion.setAttribute(
+			'aria-label',
+			_('Copy all version information in English'),
```

Comment:
Could we use variable to store the string and reuse it in many places?
`const label = _(....); `  then just use label (it is used here and one line below)

problem I want to avoid is: someone will modify one string but not other.. and we will need 2 translations for very similar string :)